CSI_Greenville_HIST8510_Final_Project-AR.qmd

```diff
@@ -0,0 +1,390 @@
+---
+title: "CSI:Dixie Greenville"
+author: "Candy Boatwright"
+format: html
+---
+
+```{r}
+#load libraries
+library(tidytext)
+library(tidyverse)
+library(readtext)
+library(tm)
+```
+
+Stephen Berry in his 2022 book, Count The Dead, addresses the rise of the modern state and its correlation with the quantification of death. During the legislative session of 1849 a tremendous reform was passed through the United States Congress, almost by accident. As the last bill before the session adjourned for the year it received almost no review and was passed as speedily as possible.
+
+The reform was the newly appointed U.S. Census Chair's, Joseph C. G. Kennedy, implementation of a census questionnaire to be administered by federal agents traveling throughout the country. The questionnaire contained six schedules, the second was to record information on the enslaved persons living on a property including their names and place of birth. While Congress, especially those from southern states, beat the life out of Schedule 2, what they failed to act on was Schedule 3 which contained the questions to be asked about who had died on a property over the preceding year and what had caused their death. So, the first mortality census was born. (Berry, Count The Dead, 18-21)
+
+The CSI:Dixie project produced by eHistory at the University of Georgia is based on coroners reports in the South primarily from the 19th century. Ten county datasets from South Carolina counties have been created to date. In this project, I begin by examining the Greenville County dataset to understand the implications of the information from the coroners' inquests and then expand my focus to include Anderson, Chesterfield and Horry counties in order to examine regional similarities and differences. Through this examination the impact of the early economic development of these counties resulting in regional variances on public health becomes apparent.
+
+```{r}
+#load CSI:Dixie Greenville County data
+CSI.GVL <- read.csv("https://docs.google.com/spreadsheets/d/e/2PACX-1vQzO3uN1w0-A_rN3bHuBWDr1iQe9hPZ68u7igu-I8wWDEpHSWH5T3AQFPvaa-VMSbuB8Z8X5wxSjmA6/pub?output=csv") 
```

Comment:
Always download your data and store it in your GitHub repo. You can load it in with the ```read.csv()``` function.